modules/microshift-accessing-cluster-remotely.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift_install/microshift-install-rpm.adoc
+// microshift/microshift_install/microshift-embed-in-rpm-ostree.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-microshift-cluster-remotely_{context}"]
+= Accessing the {product-title} cluster remotely
+
+Use the following procedure to access the {product-title} cluster from a remote workstation by using a `kubeconfig` file.
+
+* The `user@workstation` login is used to access the host machine remotely. The `<user>` value in the procedure is the name of the user that `user@workstation` logs in with to the {product-title} host.
+
+.Prerequisites
+
+* You have installed the the `oc` binary.
+
+* The `@user@microshift` has opened the firewall from the local host.
+
+.Procedure
+
+. As `user@workstation`, create a `~/.kube/` folder if your {op-system} machine does not have one by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+[user@workstation]$ mkdir -p ~/.kube/
+----
+
+. As `user@workstation`, set a variable for the hostname of your {product-title} host by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+[user@workstation]$ MICROSHIFT_MACHINE=<name or IP address of {product-title} machine>
+----
+
+. As `user@workstation`, copy the generated `kubeconfig` file that contains the host name or IP address you want to connect with from the {op-system} machine running {product-title} to your local machine by running the following command:
++
+[source,terminal]
+----
+[user@workstation]$ ssh <user>@$MICROSHIFT_MACHINE "sudo cat /var/lib/microshift/resources/kubeadmin/$MICROSHIFT_MACHINE/kubeconfig" > ~/.kube/config
+----
+
+. Update the permissions on your `~/.kube/config` file by running the following command:
++
+[source,terminal]
+----
+$ chmod go-r ~/.kube/config
```

Comment:
Is this command also run as the `[user@workstation]`? If I understand correctly, the ssh command in the previous step is writing to the `~/.kube/config` file on the `[user@workstation]` machine. 

Can we specify which user on which machine for this step as you have done with the other steps in this procedure?